README.md

```diff
@@ -103,6 +110,20 @@ Test results: "test_array_utils.cdc"
 - PASS: testMap
 - PASS: testMapStrings
 - PASS: testReduce
+Coverage: 90.6% of statements
+```
+
+Look at the files `ArrayUtils.cdc` (smart contract) and `test_array_utils.cdc` (tests for the smart contract). For the `ArrayUtils.range` method, we have omitted in purpose the branch where `start > end`. It is left as an exercise to the reader. Look at the comment on line 25 in `test_array_utils.cdc`.
```

Comment:
```suggestion
Look at the files `ArrayUtils.cdc` (smart contract) and `test_array_utils.cdc` (tests for the smart contract). 
For the `ArrayUtils.range` method, we have omitted the code branch where `start > end` on purpose. It is left as an exercise for the reader. Look at the comment on line 25 in `test_array_utils.cdc`.
```

Reply:
Updated in https://github.com/Build-Squad/flow-code-coverage/pull/1/commits/4fd71019aea467b88db1ed29592dc79b2eca7176